docs/spec/operators/filter.md

```diff
@@ -31,5 +31,8 @@ The filter syntax supports the following operations:
 - `field<value`: Matches if the content of `field` is less than `value`.
 - `field~value`: Matches if the content of `field` matches the regular expression `value`. See [RE2 Syntax](https://github.com/google/re2/wiki/Syntax) for more details.
 ```
+### `multiple filters`
+
+You can specify multiple filters by separating them with a comma. The filter `field1==value1,field2==value2` will match only events where `field1` equals `value1` and `field2` equals `value2`.
```

Comment:
This needs to tell how to escape commas if the "value" string contains a comma.

Reply:
Great point. Looking at this: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/operators/filter/filter.go#L88 I assume we don't have a way to escape comma in the value. Let me open a separate PR for it!

Reply:
I opened the PR: https://github.com/inspektor-gadget/inspektor-gadget/pull/3907